client/blocks/reader-featured-image/index.jsx

```diff
@@ -10,15 +10,13 @@ import { noop } from 'lodash';
 import cssSafeUrl from 'lib/css-safe-url';
 import resizeImageUrl from 'lib/resize-image-url';
 
-const FEATURED_IMAGE_WIDTH = 250;
-
-const ReaderFeaturedImage = ( { imageUrl, href, children, onClick } ) => {
+const ReaderFeaturedImage = ( { imageUrl, imageWidth, href, children, onClick } ) => {
```

Comment:
Allow an `imageWidth` to be provided (we want smaller images for combined card posts).